basic_bootloader/src/bootloader/runner.rs

```diff
@@ -961,20 +962,45 @@ where
     S::IO: IOSubsystemExt,
 {
     // IO will follow the rules of the CALLER (`initial_ee_version`) here to charge for execution
-    let account_properties = match system.io.read_account_properties(
-        ee_version,
-        resources,
-        &call_request.callee,
-        AccountDataRequest::empty()
-            .with_ee_version()
-            .with_unpadded_code_len()
-            .with_artifacts_len()
-            .with_bytecode()
-            .with_nonce()
-            .with_nominal_token_balance()
-            .with_code_version(),
-    ) {
-        Ok(account_properties) => account_properties,
+    let (account_properties, delegate_properties) = match system
+        .io
+        .read_account_properties(
+            ee_version,
+            resources,
+            &call_request.callee,
+            AccountDataRequest::empty()
+                .with_ee_version()
+                .with_unpadded_code_len()
+                .with_artifacts_len()
+                .with_bytecode()
```

Comment:
I suggest to add a comment why we request a bytecode even if this account is delegated

Reply:
Sure